cmd/car/root.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	carv2 "github.com/ipld/go-car/v2"
+	"github.com/ipld/go-car/v2/blockstore"
+	"github.com/urfave/cli/v2"
+)
+
+// CarRoot prints the root CID in a car
+func CarRoot(c *cli.Context) (err error) {
+	if c.Args().Len() >= 1 {
+		bs, err := blockstore.OpenReadOnly(c.Args().First())
```

Comment:
I think it would be nicer to use NewBlockReader for both; it does less work than opening a blockstore, as that often does extras we don't need like loading the index into memory. The only difference is that you need to do an `os.Open` when given a filename.